logger/middleware.py

```diff
@@ -0,0 +1,7 @@
+from .models import Log
+
+
+class LogRequestMiddleware:
+    def process_request(self, request):
+        if request.user.is_authenticated():
+            Log.objects.from_request(request)
```

Comment:
Perhaps this should be called `LogAuthenticatedRequestsMiddleware`... I think the fact that it does not log anonymous requests is peculiar to our current use-case, rather than the expected use of this lib.